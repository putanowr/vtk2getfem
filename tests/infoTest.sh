#!/bin/sh

PROG=$1
DATADIR=$2

$PROG $DATADIR/data/simple_ugrid.vtk -i -s > infoTest.out 2>&1

if [ ! -x $PROG ]; then
  echo "$PROG does not exesits"
fi 	
if cmp -s infoTest.out $DATADIR/results/infoTest.out; then
  exit 0
else
  echo "ACTUAL versu EXPECTED diff"
  diff infoTest.out $DATADIR/results/infoTest.out
  exit 1
fi
