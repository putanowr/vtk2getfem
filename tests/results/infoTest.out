Number of nodes: 8
